src/Blazored.Modal/Services/IModalService.cs

```diff
@@ -76,5 +76,18 @@ public interface IModalService
         /// <param name="parameters">Key/Value collection of parameters to pass to component being displayed.</param>
         /// <param name="options">Options to configure the modal.</param>
         ModalReference Show(Type component, string title, ModalParameters parameters, ModalOptions options);
+
+        /// <summary>
+        /// Closes the <paramref name="modal"/>
+        /// </summary>
+        /// <param name="modal">The <paramref name="modal"/> of the modal to close.</param>
+        void Close(ModalReference modal);
+
+        /// <summary>
+        /// Closes the <paramref name="modal"/>
+        /// </summary>
+        /// <param name="modal">The <paramref name="modal"/> of the modal to close.</param>
+        /// <param name="result">The result of the modal</param>
+        void Close(ModalReference modal, ModalResult result);

```

Comment:
I don't think we should add these methods to the interface, I think it would be better to keep developers working with the `ModalReference` only. What do you think?

Reply:
I agree with your point here. The reason I added it to the interface is because than the `ModalReference` can use the interface and doesn't depend on the implementation. But looking at it again, I think there will probably not be another implementation of this interface so than I think it is better to do what you suggested.